Cargo.toml

```diff
@@ -98,6 +100,7 @@ matches = "0.1.6"
 nix = "0.11.0"
 pnet_datalink = "0.21.0"
 rand = "0.5.1"
+rbpf = { git = "https://github.com/solana-labs/rbpf" }
```

Comment:
Please file a new issue for 0.10 to track removing this github dependency

Reply:
#1423